### PR TITLE
Resolves #1041

### LINKED ISSUE #1041: -Replace parameter error - Semicolon separated not comma
-Replace
 

Specifies values for an object property that will replace the current values. Use this parameter to replace one or more values of a property that cannot be modified using a cmdlet parameter. To modify an object property, you must use the LDAP display name. You can modify more than one property by specifying a **comma-separated list**. The format for this parameter is:

-Replace @{Attribute1LDAPDisplayName=value[]**,** Attribute2LDAPDisplayName=value[]}

Should be:

Specifies values for an object property that will replace the current values. Use this parameter to replace one or more values of a property that cannot be modified using a cmdlet parameter. To modify an object property, you must use the LDAP display name. You can modify more than one property by specifying a **semicolon separated list**. The format for this parameter is:

-Replace @{Attribute1LDAPDisplayName=value[]**;** Attribute2LDAPDisplayName=value[]}

-----------------

Semicolon separated, not comma.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 30217138-eb8c-ab9a-4e11-991c6164cbd9
* Version Independent ID: 88ff510e-6402-5163-404f-e9154f59b381
* Content: [Set-ADUser (addsadministration)](https://docs.microsoft.com/en-us/powershell/module/addsadministration/set-aduser?view=win10-ps#feedback)
* Content Source: [docset/windows/addsadministration/set-aduser.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/addsadministration/set-aduser.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @coreyp-at-msft
* Microsoft Alias: **coreyp**

### PR DESCRIPTION
**Pull request by velkovb**
Change Replace and Remove parameters to match Add description as they have the same behaviour
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1041

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/addsadministration/set-aduser.md [hunk 1]
```diff
@@ -1185,10 +1185,10 @@ Accept wildcard characters: False
 Specifies that the cmdlet remove values of an object property.
 Use this parameter to remove one or more values of a property that cannot be modified using a cmdlet parameter.
 To remove an object property, you must use the LDAP display name.
-You can remove more than one property by specifying a semicolon-separated list.
+You can specify multiple values to a property by specifying a comma-separated list of values and more than one property by separating them using a semicolon.
```
Comment (JohanFreelancer9) [inline docset/windows/addsadministration/set-aduser.md]:
```suggestion
You can specify multiple values to a property by specifying a comma-separated list of values, and more than one property by separating them using a semicolon.
```

File: docset/windows/addsadministration/set-aduser.md [hunk 2]
```diff
@@ -1213,10 +1213,10 @@ Accept wildcard characters: False
 Specifies values for an object property that will replace the current values.
 Use this parameter to replace one or more values of a property that cannot be modified using a cmdlet parameter.
 To modify an object property, you must use the LDAP display name.
-You can modify more than one property by specifying a comma-separated list.
+You can specify multiple values to a property by specifying a comma-separated list of values and more than one property by separating them using a semicolon.
```
Comment (JohanFreelancer9) [inline docset/windows/addsadministration/set-aduser.md]:
```suggestion
You can specify multiple values to a property by specifying a comma-separated list of values, and more than one property by separating them using a semicolon.
```

**Comment by dariomws306** [comment]
I am good with this.

**Comment by get-itips** [comment]
Ok

**Comment by tseward** [comment]
The technical aspects are fine, however a comma is inappropriate in each of the sentences changed:

> comma-separated list of values**,** and more than

**Comment by velkovb** [comment]
Thanks for the feedback. I am not a grammar expert so whatever you guys agree on I will adjust the article.

**Comment by JohanFreelancer9** [comment]
The use of the comma in these sentences is in accordance with the Microsoft Style Guide. To quote:
"To join independent clauses with a conjunction, such as and, or, but, or so.
Example 
Select Options, and then select Enable fast saves."

I would suggest leaving them in, but I am ok with it if you want them removed.

**Comment by tseward** [comment]
If the style guide says it should be there, then do so.

**Comment by e0i** [comment]
@AndreaBarr This PR is ready to be merged. Thanks.

**Pull request merged**